stack/.github/workflows/create-release.yml

```diff
@@ -2,7 +2,7 @@ name: Create Release
 
 on:
   schedule:
-  - cron: '*/30 * * * *'  # every 30 minutes
+  - cron: '27 2,14 * * *' # daily at 02:27 and 14:27 UTC
```

Comment:
This is the only frequency change I'm wondering about, but down to try it and we can always adjust it up to be more frequent if needed